README.md

```diff
@@ -42,4 +42,4 @@ Others
 ------
 
 Other implementations for USB in Rust
-* [embassy-usb](https://github.com/embassy-rs/embassy/blob/master/embassy-usb/src/driver.rs), an async variant.
\ No newline at end of file
+* [embassy-usb](https://github.com/embassy-rs/embassy/tree/master/embassy-usb) an async variant.
```

Comment:
Should we just point to top-level embassy instead? That way we don't break links if embassy restructures code

Reply:
Good point. And I guess embassy-usb can not easily be used stand-alone in a project which is otherwise not using embassy (or can it?). So a sentence like "The [Embassy](https://embassy.dev/dev/index.html) project has an async USB stack." might also be more helpful to the reader. Should it point to the [docs](https://embassy.dev/dev/index.html) or to the [repository](https://github.com/embassy-rs/embassy)?

Reply:
I would just point to the repo and the user can go from there